renderers/video_renderer.h

```diff
@@ -71,9 +78,9 @@ typedef struct video_renderer_s {
     video_renderer_type_t type;
 } video_renderer_t;
 
-video_renderer_t *video_renderer_dummy_init(logger_t *logger, background_mode_t background_mode, bool low_latency, int rotation);
-video_renderer_t *video_renderer_rpi_init(logger_t *logger, background_mode_t background_mode, bool low_latency, int rotation);
-video_renderer_t *video_renderer_gstreamer_init(logger_t *logger, background_mode_t background_mode, bool low_latency, int rotation);
+video_renderer_t *video_renderer_dummy_init(logger_t *logger, background_mode_t background_mode, bool low_latency, int rotation, flip_mode_t flip);
```

Comment:
Future TODO: it'd be nice if all these features got put into a configuration object and only that was passed to init.

Reply:
Obviously, no need to do this now.

Reply:
I totally agree, that would also make it easier as new capabilities are added!

Reply:
+1 👍